NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -216,5 +312,238 @@ private void RenderTabScenes()
+                            ActiveTab = GetTab("GameObject").Get();
+                        }
+
+                    }
+                    else if (field.FieldType == typeof(Text))
+                    {
+                        Text txt = (Text)field.GetValue(selectedMonoBehaviour);
+                        txt.text = GUILayout.TextField(CheckValue(field, selectedMonoBehaviour), "options");
+                        RegistrateChanges(field, selectedMonoBehaviour, txt);
+                    }
+                    else if (field.FieldType == typeof(Texture) || field.FieldType == typeof(RawImage) || field.FieldType == typeof(Image))
+                    {
+                        Texture img;
+                        if (field.FieldType == typeof(RawImage))
+                        {
+                            img = ((RawImage)field.GetValue(selectedMonoBehaviour)).mainTexture;
+                        }
+                        else if (field.FieldType == typeof(Image))
+                        {
+                            img = ((Image)field.GetValue(selectedMonoBehaviour)).mainTexture;
+                        }
+                        else
+                        {
+                            img = (Texture)field.GetValue(selectedMonoBehaviour);
+                        }
+                        GUIStyle style = new GUIStyle("box");
+                        style.fixedHeight = 250 * (img.width / img.height);
+                        style.fixedWidth = 250;
+
+                        GUILayout.Box(img, style);
+                    }
+                    else
+                    {
+                        try
+                        {
+                            //Check if convert work to prevent two TextFields
+                            Convert.ChangeType(field.GetValue(selectedMonoBehaviour).ToString(), field.FieldType);
+                            RegistrateChanges(field, selectedMonoBehaviour, Convert.ChangeType(GUILayout.TextField(CheckValue(field, selectedMonoBehaviour), "options"), field.FieldType));
+                        }
+                        catch
+                        {
+                            GUILayout.TextField("Not implemented yet", "options");
+                        }
+                    }
+                }
+            }
+        }
+
+        private void RegistrateChanges(FieldInfo field, Component component, object value)
+        {
+            if (editMode)
+            {
+                if (GetFromField(field) == null)
+                {
+                    DebuggerAction action = new DebuggerAction(component, field, component, value);
+                    actionList.Add(action);
+                }
+                else
+                {
+                    GetFromField(field).Value = value;
+                }
+            }
+        }
+
+        private DebuggerAction GetFromField(FieldInfo field)
```

Comment:
This method would be a good candidate for a `private DebuggerAction GetFromField(FieldInfo field) {}` method inside the `DebuggerActions` class (if you created it).